README.md

```diff
@@ -658,6 +658,14 @@ Default: empty
 
 Specify the EC2 endpoint to use. This is useful if you are using a custom endpoint for EC2. For example, if you are using a proxy for EC2, you can set this to the proxy endpoint. Any kind of URL or IP address is valid such as `https://localhost:8080` or `http://ec2.us-west-2.customaws.com`. If this is not set, the default EC2 endpoint will be used.
 
+#### `DISABLE_LEAKED_ENI_CLEANUP` (v1.13.0+)
+
+Type: Boolean as a String
+
+Default: `false`
+
+On IPv4 clusters, IPAMD schedules an hourly background task per node that cleans up leaked ENIs. Setting this environment variable to `true` disables that job. The primary motivation to disable this task is to decrease the amount of EC2 API calls made from each node.
```

Comment:
Nit: Lets add a note that this has to be considered carefully since this might lead to leaked ENIs (https://github.com/aws/amazon-vpc-cni-k8s/issues/1223) and would need manual cleanup..

Reply:
added in latest revision